engine/src/flutter/impeller/compiler/compiler.cc

```diff
@@ -32,6 +32,8 @@ constexpr const char* kEGLImageExternalExtension300 =
     "GL_OES_EGL_image_external_essl3";
 }  // namespace
 
+static const uint32_t kMaxUniformBufferSize = 6208;
```

Comment:
The theory is that this isn't sufficient, but this is the max size right now and it's used by `conical_gradient_uniform_fill.frag`